hikari/impl/entity_factory.py

```diff
@@ -1173,6 +1173,9 @@ def deserialize_guild_stage_channel(
             snowflakes.Snowflake(overwrite["id"]): self.deserialize_permission_overwrite(overwrite)
             for overwrite in payload["permission_overwrites"]
         }
+        last_message_id = (
+            snowflakes.Snowflake(msg_id) if (msg_id := payload.get("last_message_id")) is not None else None
+        )
```

Comment:
```suggestion
        last_message_id = snowflakes.Snowflake(payload["last_message_id"]) if "last_message_id" in payload else None
```

Should make this a nicer one-liner

Reply:
It was at first, I dunno if black was too happy with it though

Reply:
Line length is bellow 120, so shouldn't be too many issues. Regardless, reads nicer IMO

Reply:
Okay :)